README.md

```diff
@@ -80,12 +80,14 @@ The script `clean-old-results` can be used to get rid of undesired old results (
 
 ## How to contribute
 
-It is easy: just create a Pull Request with your contribution, and make sure that the unit tests are passing.
+It is easy: 
+- Make sure you have Git commit signing enabled. If you are not doing it already, check out the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
```

Comment:
```suggestion
It is easy:

- Make sure you have Git commit signing enabled. If you are not doing it already, check out the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
```
Trailing whitespace and lists should be surrounded by blank lines.

Reply:
https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md032.md :wink:

Reply:
*sigh*, so it's about some parses not working correctly:
> Rationale: Aside from aesthetic reasons, some parsers, including kramdown, will not parse lists that don't have blank lines before and after them.

The aesthetic reasons could be discussed here, as the paragraph is the explanation for the list, but OK.